cds_ils/cli.py

```diff
@@ -294,6 +292,32 @@ def create_demo_items(items_path):
 @with_appcontext
 def create_loan(user_email, is_past_loan):
     """Create a loan."""
+    ongoing_loan_item_pid = "vgrh9-jvj8E"
```

Comment:
please leave a comment that this part is hardcoded in demo_data because it is confusing from where the IDs are coming from